src/late_multicellular_stage/editor/MetaballEditorComponentBase.cs

```diff
@@ -575,14 +575,14 @@ protected void OnMetaballEditorMouseEntered()
         UpdateMutationPointsBar();
     }
 
-    /// <inheritdoc cref="HexEditorComponentBase{TEditor,TCombinedAction,TAction,THexMove}.OnHexEditorMouseExited"/>
+    /// <inheritdoc cref="HexEditorComponentBase{TEditor,TCombined,TAction,THexMove,TContext}.OnHexEditorMouseExited"/>
```

Comment:
There's still changes in this file, so I think you might as well put the correct stuff here (note that due to the line length, there's a need to disable the too long line check for these few lines. You can check my open refactor PR where I did this. Or you could just revert all changes to this file. I think the CI checks will stop checking this file once you make one extra commit after the one reverting the changes in this file.